### ISSUE TITLE
g-link consider attributes `href`

### ISSUE THREAD

**Issue by tyankatsu0105** [gridsome/eslint-plugin-gridsome#34]
**Please write rule name what you want to improve:**
require-g-link-to

**How are the rules improved?:**
Gridsome's `g-link` can use `href`.
```vue
<g-link href="http://www.example.com">sample</g-link>
👇
<a href="http://www.example.com" target="_blank" rel="noopener">sample</a>
```

Now g-link requires `to`. However if it has `href`, should work g-link.

**Provide 2-3 code examples that this rule should warn about:**

```vue
// All correct
<template>
  <div>
    <g-link to="/" class="foo">link is here</g-link>
    <g-link href="http://www.example.com" class="foo">link is here</g-link>
  </div>
</template>

```

**Additional context**
